fetch.bs

```diff
@@ -6710,6 +6752,417 @@ agent's <a>CORS-preflight cache</a> for which there is a <a>cache entry match</a
+ <li><code>https://ok.example.com/back</code> did not reserve 64 kibibytes, because it navigated back to <a for=/>top-level traversable</a>'s origin.
+ <li><p>640 - 128 - 64 - 64 = 384 kibibytes.
+</ul>
+</div>
+
+
+<p>This specification defined a <a>policy-controlled feature</a> identified by the string
+"<dfn for=PermissionsPolicy enum-value>deferred-fetch</dfn>". Its
+<a for="policy-controlled feature">default allowlist</a> is "<code>self</code>".
+
+<p>This specification defined a <a>policy-controlled feature</a> identified by the string
+"<dfn for=PermissionsPolicy enum-value>deferred-fetch-minimal</dfn>". Its
+<a for="policy-controlled feature">default allowlist</a> is "<code>*</code>".
+
+<p>The <dfn>optional nested document deferred-fetch quota</dfn> is 64 kibibytes.
+<p>The <dfn>minimal nested document deferred-fetch quota</dfn> is 8 kibibytes.
+<p>The <dfn>max containers with minimal quota</dfn> is 16.
+
+
+<div algorithm>
+<p>To get the <dfn>available deferred-fetch quota</dfn> given a {{Document}}
+<var>requestClientDocument</var> and an <a for=/>origin</a>-or-null <var>origin</var>:
+
+<ol>
+ <li><p>Let <var>quota</var> be 0.
+
+ <li><p>Let <var>quotaForOrigin</var> be 64 kibibytes.
+
+ <li>
+  <p>For each <var>otherNavigable</var> of <var>requestClientDocument</var>'s <a>node navigable</a>'s
+  <a for=navigable>top-level traversable</a>'s <a>inclusive descendant navigables</a>:
+
+  <p class=note>This algorithm iterates over the entire navigable tree. It accumulates quota from
+  the <a for=/>top-level traversable</a>, and from nested documents who inherit quota. Subsequently,
+  it subtracts the quota delegated to cross-origin nested documents, as well as quota
+  spent on pending deferred fetch requests.
+
+  <ol>
+   <li><p>Let <var>otherDocument</var> be <var>otherNavigable</var>'s <a>active document</a>.
+   <li><p>Let <var>otherContainer</var> be <var>otherNavigable</var>'s <a>navigable container</a>.
+
+   <li>
+    <p>If <var>requestClientDocument</var>'s <a for=Document>origin</a> is <a>same origin</a> with
+    <var>otherDocument</var>'s <a for=Document>origin</a>, then:
+    <ol>
+     <li>
+      <p>If <var>otherContainer</var> is null, then:
+
+      <p class=note>Accumulate the <a for=/>top-level traversable</a>'s initial quota.
+
+      <ol>
+       <li><p>If <var>otherDocument</var> is not <a>allowed to use</a> the
+       <a>policy-controlled feature</a> "{{PermissionsPolicy/deferred-fetch}}", then return 0.
+
+       <li><p>Assert: <var>quota</var> is 0.
+
+       <li>
+        <p>Set <var>quota</var> be 640 kibibytes.
+        <p class="note allow-2119">640kb should be enough for everyone.
+
+       <li><p>If <var>otherDocument</var> is <a>allowed to use</a> the
+       <a>policy-controlled feature</a> "{{PermissionsPolicy/deferred-fetch-minimal}}", then
+       decrement <var>quota</var> by <a>max containers with minimal quota</a>, multiplied by
```

Comment:
```
then decrement quota by max containers with minimal quota, multiplied by minimal quota.
```

Does this just mean `quota -= 16 * 8kb`?

Reply:
Yes